HeartBeat.php

```diff
@@ -123,11 +131,24 @@ private function getInterval(){
     private function actionBeat()
     {
         // Access the session to make sure it stays alive
-        $iSurveyId=Yii::app()->session['LEMsid'];
+        $iSurveyId = Yii::app()->session['LEMsid'];
         // If LS debug is set : show the survey id
-        if(App()->getConfig('debug'))
-        {
+        if(App()->getConfig('debug')) {
             echo $iSurveyId;
         }
     }
+
+    /**
+    * @see parent::gT for LimeSurvey 3.0
+    * @todo : add translation file
+    * deactivate function since there are no translation currently
+    */
+    private function _translate($sToTranslate, $sEscapeMode = 'unescaped', $sLanguage = null)
+    {
+
+        //~ if(is_callable($this, 'gT')) {
+        //~     return $this->gT($sToTranslate,$sEscapeMode,$sLanguage);
+        //~ }
```

Comment:
Do we need this commented code here ?

Reply:
I coment since there are no po or mo files currently :)

Reply:
I add french translation next week :)